### ISSUE TITLE
Full statistic on a new tab page

### ISSUE THREAD

**Issue by SergeyZhukovsky** [brave/browser-android-tabs#325]
Show full blocking statistic on new tab page
1. Make a new tab page view exactly the same for all search engines. It is different now only for Google search engine.
2. Create a total statistic.
![android-shield-stats](https://user-images.githubusercontent.com/12011303/28791988-38cbb432-75fc-11e7-9284-56f5817b9f16.png)

**Comment by srirambv** [comment]
Awesome :+1:

**Comment by avently** [comment]
@SergeyZhukovsky hm, do you think someone care about this stats? Best way to show  that app works is to hide ads, remove trackers, etc.  but not show the stats. 
Anyway, start page is not a good place for that information. Maybe settings can get this role.

**Comment by SergeyZhukovsky** [comment]
Yes, we think so. We have that stat on our desktop/laptop and iOS versions. People always keep asking us when does it come on Android.

**Comment by avently** [comment]
@SergeyZhukovsky hm, ok. If people need it, I can hide this view with my build for me.
